client/plots/profilePlot.js

```diff
@@ -6,6 +6,7 @@ import { select } from 'd3-selection'
 import { getSampleFilter } from '../mass/groups.js'
 import { Menu } from '#dom/menu'
 import { icons as icon_functions } from '#dom/control.icons'
+import { config } from 'process'
```

Comment:
what's this for?

Reply:
It got introduced by mistake by VS Code autocomplete. Removed now.